web/forms.py

```
@@ -1033,12 +1033,24 @@ class Meta:
+            "slug": forms.TextInput(
+                attrs={
+                    "class": "w-full border-gray-300 dark:border-gray-600 rounded p-2 bg-gray-200 cursor-not-allowed",
+                    "readonly": "readonly",
+                }
+            ),
         }
         help_texts = {
             "icon": "Enter a Font Awesome icon class (e.g., fa-folder, fa-book, fa-code)",
         }
 
+        def clean(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `clean`

(ANN201)